webpack.config.js

```diff
@@ -39,21 +39,29 @@ function convertEmojiToolkitCodePointToChar(codePoint) {
 
 function jsonTransform(emojis) {
   const result = {};
+
   for (const emojiData of Object.values(emojis)) {
     const char = convertEmojiToolkitCodePointToChar(emojiData.code_points.fully_qualified);
     const data = {
       char,
       slug: emojiData.shortname.replace(/:/g, ''),
+      category: emojiData.category,
+      isAlternative: false,
```

Comment:
so we can save a little extra space in memory, can we only specify alternatives if they are true?